imaging/bragg-edge-imaging-2D.ipynb

```diff
@@ -152,7 +152,7 @@
     "\n",
     "# Chopper cascade\n",
     "beamline = imaging.make_beamline()\n",
-    "ds.coords['choppers'] = sc.Variable(beamline[\"choppers\"])\n",
+    "ds.coords['choppers'] = sc.scalar(beamline[\"choppers\"])\n",
```

Comment:
if we are moving to more modern `scalar` we should probably move all dimensional variables to be `array` too.